RijsMuseum/app/src/main/java/com/example/rijsmuseum/adapter/RecyclerAdapter.kt

```diff
@@ -7,17 +7,17 @@ import android.widget.ImageView
 import android.widget.TextView
 import androidx.recyclerview.widget.RecyclerView
 import com.bumptech.glide.Glide
-import com.example.network.models.Collections
+import com.example.network.resource.models.CollectionsResource
 import com.example.rijsmuseum.R
 
 class RecyclerAdapter : RecyclerView.Adapter<RecyclerAdapter.ViewHolder>() {
     //Collections ArtObjects MutableList
-    private val cList: MutableList<Collections.ArtObject> = mutableListOf()
+    private val cList: MutableList<CollectionsResource.ArtObjectResource> = mutableListOf()
```

Comment:
I assume the resource is the raw object coming directly from the request. If that is the case, then you shouldn't use that here. Instead you should use the model from the domain package.